lib/config/v2/elevator.ex

```diff
@@ -3,14 +3,23 @@ defmodule ScreensConfig.V2.Elevator do
 
   alias ScreensConfig.V2.EvergreenContentItem
 
+  @type arrow_direction :: :n | :s | :e | :w
+
   @type t :: %__MODULE__{
           elevator_id: String.t(),
-          evergreen_content: list(EvergreenContentItem.t())
+          evergreen_content: list(EvergreenContentItem.t()),
+          alternate_direction_text: String.t(),
+          accessible_path_image_url: String.t(),
+          accessible_path_direction_arrow: arrow_direction()
```

Comment:
I can go ahead and stub them out. I think it's important for these to be required in the future because without them, the screens would be mostly blank if the current elevator is closed.